docs/logging.md

```diff
@@ -0,0 +1,34 @@
+# Logging
+
+For logging use the `Log` struct provided in the `log` package (see `pkg/log`).
+That struct offers several functions for providing logs with different log levels, verbosity and content.
+
+## Log Levels and Verbosity
+
+The Log struct offers several functions for logging with a decent log level: `Debug(msg)`, `Info(msg)`, `Warning(msg)`, `Error(msg)`, `Critical(msg)`.
```

Comment:
`Debug` should be dropped. We have verbosity in combination with Info. glog does not have it either and I personally find it confusing when to use what or what the difference should be (e.g at which verbosity level does debug start?).

Reply:
Completely agree that debug is confusing as it is now. Will remove it.

Reply:
@rmohr should I also remove it from code right now, and replace existing calls to it with `V(4).Info(...)`?

Reply:
Would be great.